kani-compiler/src/codegen_cprover_gotoc/utils/names.rs

```diff
@@ -51,12 +51,9 @@ impl<'tcx> GotocCtx<'tcx> {
     }
 
     /// Return the mangled name to be used in the symbol table.
-    ///
-    /// We special case main function in order to support `--function main`.
-    // TODO: Get rid of this: https://github.com/model-checking/kani/issues/2129
+    #[inline(always)]
     pub fn symbol_name_stable(&self, instance: Instance) -> String {
-        let pretty = instance.name();
-        if pretty == "main" { pretty } else { instance.mangled_name() }
+        instance.mangled_name()
```

Comment:
Might as well just remove this function and call `instance.mangled_name()` wherever you need it.

Reply:
How much I love my IDE. That was a one click change. :smile: